src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -44,19 +44,14 @@ export const getters: GetterTree<ICustomFlowState, IRootState> = {
 
     return Object.keys(currentBlock.config.choices).reduce((memo, choiceKey): {[key: string]: IInflatedChoicesInterface} => {
       const resourceUuid = currentBlock.config.choices[choiceKey]
+      const exit = findExitFromResourceUuid(resourceUuid, currentBlock, !getters.isExitsBranchingSegregated)
       memo[choiceKey] = {
-        exit: getters.blockExitFromResourceUuid(resourceUuid),
+        exit,
         resource: rootGetters['flow/resourcesByUuid'][resourceUuid]
       }
       return memo
     }, choices)
   },
-  blockExitFromResourceUuid: (state, getters, rootState, rootGetters) => (resourceUuid: string): IBlockExit => {
```

Comment:
took the opportunity to remove dynamic getter in favor of `findExitFromResourceUuid` function